_sass/base.scss

```diff
@@ -4662,10 +4666,10 @@ sup {
     background-color: $white;
     border-top-left-radius: 5px;
     overflow: hidden;
-    height: 100%;
+    height: 100%; 
     img {
       max-width: 140%;
-      width: 140%;
+      width: 100%;
```

Comment:
Images don't fill up the entire height

Images on this branch|Original
----------------------|--------
![image](https://user-images.githubusercontent.com/12103383/54742903-a4bf3100-4be8-11e9-92e1-778d2e1a8dc0.png)|![image](https://user-images.githubusercontent.com/12103383/54742922-b43e7a00-4be8-11e9-82a2-bd960601934f.png)

Setting width to 140% should fix this

Reply:
@ramyaragupathy  changes done.